src/index.html

```diff
@@ -1,13 +1,402 @@
 <!DOCTYPE html>
-<html lang="en">
+<html  lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>Miami | Condo Kings</title>
```

Comment:
No need to separate this phrase, it's may confuse the user, he might think that this is Miami City's website and he's on the Condo Kings page. Whilst you're making a landing page for Miami Condo Kings company.
```suggestion
    <title>Miami Condo Kings</title>
```